### DISCUSSION TITLE
Error in setting up the domain with Github Pages

### DISCUSSION THREAD

**Discussion by xylium117** [is-a-dev/register#8009] [Q&A] [answered]
I am trying to publish a vite.js server webpage on Github Pages. The default .github.io domain is working, but when I am trying to put my domain name "ayushmansarkar.is-a.dev" in the custom domains field, it fails the DNS checks.

When I go to http://ayushmansarkar.is-a.dev/ it tells me to sign in to FTP, but I don't know my credentials (password). It says "Set by User".

**Comment by wdhdev** [comment, marked as answer]
You are using is-a.dev hosting, which is why your DNS checks are failing. You need to change your records to point to GitHub Pages. Our [docs](https://www.is-a.dev/docs) can help you.
